src/interface/meta.thrift

```diff
@@ -89,6 +101,21 @@ struct EdgeItem {
     4: common.Schema        schema,
 }
 
+
+
+struct UserItem {
+    1: string account,
+    2: string first_name,
```

Comment:
Do we need first_name, last_name?   
Why not put password into this structure?

Reply:
Thanks for your review @dangleptr , here are the answers to each question:
1: About first_name and last_name, Discussed with  @sherman-the-tank in issue #190 , Need to add more properties to a user, such as email address, first name, last name, etc.
2:About password is not contained in the structure UserItem，the reason is functional difference is between password and other user attributes. password is only used when connecting or changing password, so password is not normal user attributes. Also, fast read and write do not require serialization or deserialization of structures.

Reply:
> 1: About first_name and last_name, Discussed with @sherman-the-tank in issue #190 , Need to add more properties to a user, such as email address, first name, last name, etc.

We'd better just saved the information our system used.  Just supposed that, when you create a account,  the system ask you to fill your personal information,  is it reasonable?

> 2:About password is not contained in the structure UserItem，the reason is functional difference is between password and other user attributes. password is only used when connecting or changing password, so password is not normal user attributes. Also, fast read and write do not require serialization or deserialization of structures.

Make sense.